symfony/client/src/components/dialogs/DeleteConfirmationDialog.vue

```diff
@@ -0,0 +1,81 @@
+<template>
```

Comment:
Better to OS code header for all VUE files

Reply:
OS code added